aikau/src/main/resources/alfresco/lists/views/HtmlListView.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * Copyright (C) 2005-2015 Alfresco Software Limited.
+ *
+ * This file is part of Alfresco
+ *
+ * Alfresco is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Alfresco is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/**
+ * 
+ * @module alfresco/lists/views/HtmlListView
+ * @extends module:alfresco/lists/views/AlfListView
+ * @author Dave Draper
+ */
+define(["dojo/_base/declare",
+        "alfresco/lists/views/AlfListView",
+        "dojo/text!./templates/HtmlListView.html",
+        "alfresco/lists/views/ListRenderer",
+        "dojo/text!./templates/HtmlListViewRenderer.html",
+        "dojo/_base/lang",
+        "dojo/_base/array",
+        "dojo/dom-construct"], 
+        function(declare, AlfListView, template, ListRenderer, rendererTemplate, lang, array, domConstruct) {
+   
+   var Renderer = declare([ListRenderer], {
+
+      templateString: rendererTemplate,
+
+      renderData: function alfresco_dnd_DragAndDropItems__renderData() {
+         array.forEach(this.items, function(item) {
+            var property = lang.getObject(this.propertyToRender, false, item);
```

Comment:
We iterate over the items here that are set in the main HtmlListView widget - all of the data handling, etc. is handled by the parent module, we're just overriding what to actually do with each item. I've used a closure here - but once again - when this is finalized in a future sprint it would be a bound function.